packages/web/templates/core/upstreams.conf.template

```diff
@@ -1,6 +1,6 @@
   # DNS resolution works at startup via `depends_on` for `web` service in compose.yaml.
   upstream api_server {
-    server api:3000;
+    server ${UPSTREAM_API_HOST}:3000;
```

Comment:
In case I will use [`awsvpc` network mode](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-networking-awsvpc.html) (which allows containers to communicate with `localhost`). This happens when running [nginx as an aws copilot sidecar](https://github.com/aws/copilot-cli/discussions/2674).